dist/spec/payid-easy-checkout.txt

```diff
@@ -0,0 +1,728 @@
+3.  PayID Easy Checkout Protocol
+
+   The PayID Easy Checkout Protocol can be used to facilitate an end-to-
+   end checkout flow between a payment recipient, such as an online
+   merchant, and a sending client, such as a wallet.
+
+   The protocol is comprised of two parts:
+
+
+
+
+Kramer, et al.          Expires January 24, 2021                [Page 3]
+
+Internet-Draft   Draft 1 - PayID Easy Checkout Protocol        July 2020
+
+
+   1.  PayID Easy Checkout Discovery
+
+   2.  PayID Easy Checkout URL Assembly
+
+3.1.  PayID Easy Checkout Discovery
+
+   PayID Easy Checkout Discovery extends [PAYID-DISCOVERY] by defining a
+   new link in the PayID metadata JRD returned by a PayID Discovery
+   query.  This link, defined in the JRD section (Section 4) of this
+   specification, includes the PayID Easy Checkout URL representing a
+   resource on the wallet which can be used to complete a payment.
+
+   Payment receivers who wish to initiate an Easy Checkout flow MUST
+   query the sender's PayID Discovery server to obtain a PayID Easy
+   Checkout URL.  PayID servers that wish to enable PayID Easy Checkout
+   MUST include a JRD Link conforming to the definition in the JRD
+   section (Section 4) of this paper in all PayID Easy Checkout
+   Discovery responses.
+
+   Payment receivers SHOULD implement fallback measures to complete a
+   checkout flow if a user's wallet does not support PayID Easy
+   Checkout.
+
+   The following steps describe how a PayID client can query a PayID
+   server to obtain a PayID Easy Checkout URL.
+
+3.1.1.  Step 1: Assemble PayID Easy Checkout Discovery URL
+
+   The process of assembling a PayID Discovery URL is defined in section
+   4.1.1 of [PAYID-DISCOVERY], and is the same as for this protocol.
+
+3.1.2.  Step 2: Query PayID Easy Checkout Discovery URL
+
+   Querying the PayID Discovery URL is defined in section 4.1.2 of
+   [PAYID-DISCOVERY], and is performed in the same way as this protocol.
+
+   Clients SHOULD implement fallback measures to complete checkout if
+   the PayID Easy Checkout Discovery query fails.
+
+3.1.3.  Step 3: Parse PayID Easy Checkout Metadata
+
+   If PayID Easy Checkout is supported, a PayID Discovery server MUST
+   respond with an HTTP status code "200" and a JSON payload containing
+   a JSON Resource Descriptor (JRD) that contains a link conforming to
+   the JRD section (Section 4) of this document.
+
+   For example, a PayID server might respond to a PayID Discovery query
+   with the following payload:
+
+
+
+Kramer, et al.          Expires January 24, 2021                [Page 4]
+
+Internet-Draft   Draft 1 - PayID Easy Checkout Protocol        July 2020
+
+
+    {
+       "subject": "payid:alice$wallet.com",
+       "links": [
+           {
+               "rel" : "https://payid.org/ns/payid-easy-checkout/1.0",
+               "href": "https://wallet.com/checkout"
+           }
+       ]
+    }
+
+   A PayID Discovery client MUST parse this response to find the PayID
```

Comment:
PayID Discovery Client? This is the first usage of this term.

Reply:
Changed to PayID Easy Checkout Client